helm/mongodb/values.yaml

```diff
@@ -3,11 +3,11 @@ quickSetting:
   namespace: default
   env: test-rs
   image:
-    mongodb: registry.cern.ch/cmsweb/cmsmongo:HG2310a
+    mongodb: registry.cern.ch/cmsweb/cmsmongo:HG2310f-test
   pvc:
     storageClassName: cpio1
     accessMode: ReadWriteMany
-    storageSize: 1Ti
+    storageSize: 1Ti       
```

Comment:
@arooshap 

Do they really need `1Ti` storageSize? Could you please confirm from WMCore team. Thanks